### PR TITLE
android-studio-full: fix changelog ordering

### PR DESCRIPTION
**Pull request by numinit**
## Description of changes

Followup to #315233 fixing the changelog ordering and moving the relnote to 24.11.

## Things done

- Built on platform(s)
  - [x] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- For non-Linux: Is sandboxing enabled in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
  - [ ] `sandbox = relaxed`
  - [ ] `sandbox = true`
- [ ] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md#package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [ ] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [24.11 Release Notes](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2411.section.md) (or backporting [23.11](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2311.section.md) and [24.05](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2405.section.md) Release notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

---

Add a :+1: [reaction] to [pull requests you find important].

[reaction]: https://github.blog/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/
[pull requests you find important]: https://github.com/NixOS/nixpkgs/pulls?q=is%3Aopen+sort%3Areactions-%2B1-desc

### CODE REVIEW

**Review by numinit — COMMENTED** [review]
gtest is documented at https://github.com/NixOS/nixpkgs/pull/320194/files#diff-9c253a17d297f3da51e6ef73a3cb898d879ccf8b0c8613199770917e9387d92cL311 and became part of the android-studio-full PR due to a bad merge.

**Review by drupol — APPROVED** [review]

**Review by JohnRTitor — CHANGES_REQUESTED** [review]
https://github.com/NixOS/nixpkgs/pull/320193#pullrequestreview-2121130123

File: nixos/doc/manual/release-notes/rl-2405.section.md [hunk 1]
```diff
@@ -534,10 +534,6 @@ Use `services.pipewire.extraConfig` or `services.pipewire.configPackages` for Pi
   - to get previous behaviour of upstream defaults, set it to `null`
   - default value has changed from `[]` to `null`, in order to preserve default behaviour
 
-- `androidenv.androidPkgs_9_0` has been removed, and replaced with `androidenv.androidPkgs` for a more complete Android SDK including support for Android 9 and later.
-
-- `gtest` package has been updated past v1.13.0, which requires C++14 or higher.
```
Comment (JohnRTitor) [inline nixos/doc/manual/release-notes/rl-2405.section.md:539]: Looks like this was accidentally removed?

**Comment by drupol** [comment]
> gtest is documented at https://github.com/NixOS/nixpkgs/pull/320194/files#diff-9c253a17d297f3da51e6ef73a3cb898d879ccf8b0c8613199770917e9387d92cL311 and became part of the android-studio-full PR due to a bad merge.

^^

File: nixos/doc/manual/release-notes/rl-2405.section.md (hunk 1 above)
Comment (numinit) [reply nixos/doc/manual/release-notes/rl-2405.section.md:539]: That gtest line was actually added by #315233. I was in the process of cleaning it up and rebasing the commit when it was merged :no_mouth:

**Comment by JohnRTitor** [comment]
Oh.

**Review by JohnRTitor — APPROVED** [review]
Then LGTM

**Pull request merged**